manuscript/Figure3-currentcogscore-calculations.jl

```diff
@@ -47,17 +47,17 @@ ecs = Resonance.load(ECProfiles(); timepoint_metadata = seqs)
 mdata_ecs_df = sort(Resonance.comm2wide(ecs), [ :subject, :timepoint ]);
 
 ### 3.1. Hashing EC names with fixed-length alphabetic hashes due to length and special characters; storing text files for remapping.
-oldnames = names(mdata_ecs_df)[20:end]
-for oldname in names(mdata_ecs_df)[20:end]
+oldnames = names(mdata_ecs_df)[22:end]
```

Comment:
Is there any way to get the `22` programatically? This feels really brittle

Reply:
I think you might be able to do this with a regular expression. Try `names(mdata_ecs_df, r"^[A-Z][a-z]+_\w+$")`. You can use `Not` to see what this *doesn't* grab

Reply:
I think you might be able to do this with a regular expression. Try `names(mdata_ecs_df, r"^[A-Z][a-z]+_\w+$")`. You can use `Not` to see what this *doesn't* grab.